fuse_models/include/fuse_models/parameters/odometry_2d_publisher_params.h

```diff
@@ -81,6 +81,10 @@ struct Odometry2DPublisherParams : public ParameterBase
     nh.param("scale_process_noise", scale_process_noise, scale_process_noise);
     nh.param("velocity_norm_min", velocity_norm_min, velocity_norm_min);
 
+    double covariance_throttle_period_double = covariance_throttle_period.toSec();
+    fuse_core::getPositiveParam(nh, "covariance_throttle_period", covariance_throttle_period_double, false);
+    covariance_throttle_period.fromSec(covariance_throttle_period_double);
```

Comment:
I'll rebase and change this to 
```suggestion
    fuse_core::getPositiveParam(nh, "covariance_throttle_period", covariance_throttle_period, false);
```
when #204 is in.

Reply:
Done.